client/components/application-summary.js

```diff
@@ -95,6 +97,24 @@ class ApplicationSummary extends React.Component {
     }
   }
 
+  changed = (key) => {
+    const fields = this.props.fieldsBySection[key] || [];
+    if(this.props.changed.includes(key) || this.props.changed.some(k=> fields.includes(k) )) {
+      return CHANGED;
+    }
+  }
```

Comment:
@lvnhmd I think you'll also need to include looping sections - see the use of `repeats` property in the comments code

Reply:
done